lib/sorbet-rails/tasks/rails_rbi.rake

```diff
@@ -67,7 +63,8 @@ namespace :rails_rbi do
   end
 
   def copy_bundled_rbi
-    bundled_rbi_path = File.join(@@sorbet_rails_rake_dir, "..", "rbi", ".")
+    sorbet_rails_rake_dir = "#{Dir.pwd}/lib/sorbet-rails/tasks"
```

Comment:
I'm not sure this work when people install the gem & run it from rake. `Dir.pwd` will refer to the location they run the rake task, not the root directory of the gem.

Reply:
How about: sorbet_rails_rake_dir = "#{Gem::Specification.find_by_name("sorbet-rails").gem_dir}/lib/sorbet-rails/tasks"

Reply:
Let's try.